qualtran/_infra/gate_with_registers.py

```diff
@@ -549,3 +546,51 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> cirq.Circ
 
         wire_symbols[0] = self.__class__.__name__
         return cirq.CircuitDiagramInfo(wire_symbols=wire_symbols)
+
+
+def get_ctrl_system_for_single_qubit_controlled(
+    bloq: GateWithRegisters, ctrl_spec: Optional['CtrlSpec'] = None
+) -> Tuple['Bloq', 'AddControlledT']:
+    """Wrapper to correctly wire a custom implementation of a single-qubit controlled version of a gate.
+
+    For any gate that supports a single-qubit controlled version
+    with an API that has `control_val` and `control_registers`.
+    Add the `get_ctrl_system` implementation as shown below:
+
+    .. code:: python
```

Comment:
we actually render all this as markdown, so this directive won't work properly. I think the indented section should be rendered as code; you might need backticks if you want to say "python"

Reply:
oops, fixed now